setup.py

```diff
@@ -186,7 +195,7 @@ def build_extension(self, ext):
         'Source': 'https://github.com/openPMD/openPMD-api',
         'Tracker': 'https://github.com/openPMD/openPMD-api/issues',
     },
-    ext_modules=[CMakeExtension('openpmd_api_cxx')],
+    ext_modules=[CMakeExtension('openpmd_api.openpmd_api_cxx')],
```

Comment:
Playing with declaring the full sub-module here does not help either with the install pick-up-to-wheel issue.